src/components/navigation-header/types.ts

```diff
@@ -1,7 +1,5 @@
-interface NavigationHeaderItem {
+export interface NavigationHeaderItem {
```

Comment:
See comment in `src/components/navigation-header/index.tsx` that explains the changes here.